### DISCUSSION TITLE
Slideshow Widget - Text Overlay Background

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#609] [Q&A]
I'm currently trying out the slideshow widget and I'm having a problem with it. The background images must fit the theme of the website and I have good images for that. My problem... in some areas of the background images no light font is visible, in other areas of the background images no dark font is visible. So my text overlay isn't really readable. Is it possible to define a separate background for the text overlays or to darken the background images when sliding in the text overlays? Here is a similar example from a Joomlaplates template.

https://demo.joomlaplates.de/hope/

**Comment by sonvnn** [comment]
@Banuk1616 You can use custom css. Add this code 
```
.overlay-inner {
    background: green;
    padding: 30px 40px 10px;
}
```

The result here 
![Screen Shot 2024-03-19 at 09 51 57](https://github.com/templaza/astroid-framework/assets/3204103/0ebc28c7-57e9-45e0-8155-f8e9b4ed7262)

**Reply by Banuk1616** [reply]
@sonvnn 

Thank you very much, that helps me.
It would perhaps be nice if this could be set directly in the settings of the slideshow widget in the future.

**Comment by Chacapamac** [comment]
> It would perhaps be nice if this could be set directly in the settings of the slideshow widget in the future.
Make Sense!
